src/liballoc/collections/vec_deque.rs

```diff
@@ -197,14 +201,20 @@ impl<T> VecDeque<T> {
     unsafe fn wrap_copy(&self, dst: usize, src: usize, len: usize) {
         #[allow(dead_code)]
         fn diff(a: usize, b: usize) -> usize {
-            if a <= b { b - a } else { a - b }
+            if a <= b {
+                b - a
+            } else {
+                a - b
+            }
```

Comment:
Please don't mix a rustfmt of the whole file with actual changes in the same commit; it makes reviewing more difficult than it needs to be.

Reply:
Ok, I'll undo the rustfmt if I find the time.

Reply:
Done.